src/jobs/generation/MetaGenerator.groovy

```diff
@@ -294,6 +294,9 @@ repos.each { repoInfo ->
     definitionScriptSuffix = definitionScriptSuffix.replace(".groovy", "")
 
     [true, false].each { isPRTest ->
+        // In cases where we are making a PR to the utilities repo, we should run the DSL
+        // tests by default against the CI SDK from the PR.
```

Comment:
This is the case for repo list entries that are marked as DSL tests, meaning that their generation triggers should be default, but also if the repo in the repo list is the same as the SDK, we should also pull that from the PR, rather than from the default branch.